build/schema.js

```diff
@@ -170,7 +170,7 @@ module.exports = {
       description: 'uuid',
       type: 'string',
       allowEmpty: false,
-      pattern: /^[0-9a-f]{62}$/
+      pattern: /^[0-9a-f]{16,62}$/
```

Comment:
If it's 16 chars it's 8 bytes actually.
I don't mind but seems the new UUIDs are 32 hex chars long, 16 bytes, and the old were 31 bytes.
Am I wrong?